### ISSUE TITLE
Adjusting axis in johnson neyman plots?

### ISSUE THREAD

**Issue by jlhanson5** [jacob-long/jtools#15]
Hello all,

I'm loving jtools and just had a (hopefully) quick question about how to plot a three-way interaction and adjust the axis on the graph. I was using sim_slopes with jnplot = TRUE to depict my three-way interaction and region of significance... but I wondered how to adjust the axes? 

I know this can be down in the johnson_neyman function (**but** that didn't allow a three-way interaction graph from my read). Is there a simple argument to add to my sim_slopes call to change things? I tried ylim = c(-3, 3) at the end of the call... but that didn't work. Other thoughts?

Any advice is greatly appreciated!

Thanks much!
Jamie.

**Comment by jacob-long** [comment]
That's an interesting question and your read on your options is correct. I make that multi-paned Johnson-Neyman plot for 3-way interactions using the `cowplot` package, meaning the outputted plot isn't exactly a normal `ggplot`. Making changes to the individual plots unfortunately won't be particularly straightforward, but here's the strategy I'd use:

* Save sim_slopes to an object (`ss <- sim_slopes(...)`)
* In that object, you have a `jnplot` element. However, for the 3-way interaction this is the `cowplot`-assembled plot that won't respond sensibly to a `+ xlim(-3,3)` call. What we need is the `jn` element, which is a list of `johnson_neyman` objects. Maybe it's best to save that to another object for simplicity (`jns <- ss$jn`).
* For each plot in the list of `johnson_neyman` objects, we'll want to apply the `ggplot` function calls. So we might do something like this:

```r
plot_list <- list()
for (j in seq_along(jns) {
  plot_list[[j]] <- jns[[j]]$plot + xlim(-3, 3)
}
``` 

There will be some warnings about overriding existing scales, but that's nothing to worry about. Now `plot_list` is a list of the revised Johnson-Neyman plots.

* From there, you can recreate the multi-pane plot using the `cowplot` package. There might be some useful hints for how to do that in the `jtools` source code around here: https://github.com/jacob-long/jtools/blob/cb8304fc30dedc4b666ae30a96a9c43d606e2242/R/simple_slopes.R#L649

If you think the adjustment you need to make is one that many others would likely want, let me know and I can think about whether it would be appropriate to build an argument into `johnson_neyman` or `sim_slopes` to accommodate it.

**Comment by jacob-long** [comment]
This is probably fixed in v1.1.0
